### PR TITLE
Helpers 2.1

### PR DESCRIPTION
**Pull request by alexAubin**
Applied https://github.com/YunoHost/apps/pull/2407 + a bunch of tweaks for example to add a comment about `ynh_backup` behavior for `$data_dir` and `/var/log/$app`

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -47,39 +39,44 @@ ynh_systemd_action --service_name="$app" --action="stop" --log_path="/var/log/$a
 # If db_name doesn't exist, create it
 # if [ -z "$db_name" ]; then
 #     db_name=$(ynh_sanitize_dbid --db_name=$app)
-#     ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+#     ynh_app_setting_set --key=db_name --value=$db_name
 # fi
```
Comment (ericgaspar) [inline scripts/upgrade:43]: ynh_app_setting_set_default --key=db_name --value=$(ynh_sanitize_dbid --db_name=$app)

File: scripts/upgrade [hunk 2]
```diff
@@ -47,39 +39,44 @@ ynh_systemd_action --service_name="$app" --action="stop" --log_path="/var/log/$a
...
 
 # If install_dir doesn't exist, create it
 # if [ -z "$install_dir" ]; then
 #     install_dir=/var/www/$app
-#     ynh_app_setting_set --app=$app --key=install_dir --value=$install_dir
+#     ynh_app_setting_set --key=install_dir --value=$install_dir
 # fi
```
Comment (ericgaspar) [inline scripts/upgrade:49]: ynh_app_setting_set_default --key=install_dir --value="/var/www/$app"

**Comment by ericgaspar** [comment]
is `extra_php-fpm.conf` file needed?

**Comment by alexAubin** [comment]
>is extra_php-fpm.conf file needed?

Not in the vast majority of cases, it's been replaced by `ynh_app_setting_set --key=php_upload_max_filesize --value=<value>` to be added to the install and upgrade script

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**